ui.frontend/src/utils/api.types.ts

```diff
@@ -58,7 +58,7 @@ export type SelectArgument = Argument<ArgumentValue> & {
 };
 
 export type MultiSelectArgument = Argument<ArgumentValue> & {
-  options: Record<string, ArgumentValue>;
+  options: Record<string | number, ArgumentValue>;
```

Comment:
nope, the idea is to allow to define keys as nums but internally keep them strings